web/tools/webpack.config.babel.js

```diff
@@ -63,7 +63,7 @@ const generateManifest = (content: string, buildConfigName: string): string => {
 }
 
 const createConfig = (env: { config_name?: string, dev_server?: boolean, version_name?: string, commit_sha?: string } = {}) => {
-  const { config_name: buildConfigName, commit_sha: commitSha, version_name: versionName, dev_server: devServer } = env
+  let { config_name: buildConfigName, commit_sha: commitSha, version_name: versionName, dev_server: devServer } = env
```

Comment:
I'd like if this stays `const`, e.g.:
```suggestion
  let { config_name: buildConfigName, commit_sha: passedCommitSha, version_name: passedVersionName, dev_server: devServer } = env
```
And doing
```
  const versionName = passedVersionName ?? readVersionName()
  const commitSha = passedCommitSha?.substring(0, SHORT_COMMIT_SHA_LENGTH) ?? 'Commit SHA unknown'
```
below.